plugins/grpc/python/v1.62.2/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/grpc/python
+plugin_version: v1.62.2
+source_url: https://github.com/grpc/grpc
+description: Generates Python client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/python:v25.3
```

Comment:
They haven't updated to v26.x yet on the v1.62.x branch (but they have on main and v1.63.x): https://github.com/grpc/grpc/blob/96f984744fe728e196c11d33b91b022566c0d40f/bazel/grpc_deps.bzl#L263-L277

We should also keep an eye out in the next release as they've bumped Python required version to 3.8.